<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>RNA-seq analysis</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html"><img id="logo" style="width: 120px; vertical-align: middle; padding-top: 5px;" src="images/VAI_2_Line_White.png" /></a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-solid fa-house"></span>
     
    Home
  </a>
</li>
<li>
  <a href="multiqc.html">
    <span class="fa fa-square-poll-vertical"></span>
     
    MultiQC
  </a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    <span class="glyphicon glyphicon-sort"></span>
     
    DE results
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li class="dropdown-header">HTML reports</li>
    <li>
      <a href="DESeq2_trt_vs_untrt.html">trt_vs_untrt</a>
    </li>
    <li class="divider"></li>
    <li class="dropdown-header">Supplemental files</li>
    <li>
      <a href="./extras/deseq2_figures/">High-res figures</a>
    </li>
    <li>
      <a href="./extras/deseq2_tables/">DE result tables</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    <span class="glyphicon glyphicon-signal"></span>
     
    GSEA results
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li class="dropdown-header">HTML reports</li>
    <li>
      <a href="gsea_trt_vs_untrt.html">trt_vs_untrt</a>
    </li>
    <li class="divider"></li>
    <li class="dropdown-header">Supplemental files</li>
    <li>
      <a href="./extras/gsea_figures/">High-res figures</a>
    </li>
    <li>
      <a href="./extras/gsea_tables/">GSEA result tables</a>
    </li>
  </ul>
</li>
<li>
  <a href="isee.html">
    <span class="fa fa-solid fa-gem"></span>
     
    iSEE
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/vari-bbc/rnaseq_workflow">
    <span class="fa fa-github"></span>
     
    Code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">RNA-seq analysis</h1>

</div>


<p>This report summarizes the RNA-seq analysis performed using the <a
href="https://github.com/vari-bbc/rnaseq_workflow">rnaseq_workflow</a>
developed by the Bioinformatics and Biostatistics Core at Van Andel
Institute.</p>
<p>The following sections describe the experimental design, summarize
the differential expression results and analytical methods, and provide
a list of software versions and references used in the analysis.</p>
<p>For detailed results, please explore the individual tabs in the
interactive report.</p>
<p><br></p>
<div id="experimental-design" class="section level2">
<h2>Experimental design</h2>
<p>We will list sample information for the experimental design
below:</p>
<table class=" lightable-classic-2" style="font-family: &quot;Arial Narrow&quot;, &quot;Source Sans Pro&quot;, sans-serif; width: auto !important; margin-left: auto; margin-right: auto;">
<caption>
Sample information
</caption>
<thead>
<tr>
<th style="text-align:left;">
sample_name
</th>
<th style="text-align:left;">
sample
</th>
<th style="text-align:left;">
group
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
SRR1039509
</td>
<td style="text-align:left;">
SRR1039509
</td>
<td style="text-align:left;">
trt
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR1039513
</td>
<td style="text-align:left;">
SRR1039513
</td>
<td style="text-align:left;">
trt
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR1039517
</td>
<td style="text-align:left;">
SRR1039517
</td>
<td style="text-align:left;">
trt
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR1039521
</td>
<td style="text-align:left;">
SRR1039521
</td>
<td style="text-align:left;">
trt
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR1039508
</td>
<td style="text-align:left;">
SRR1039508
</td>
<td style="text-align:left;">
untrt
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR1039512
</td>
<td style="text-align:left;">
SRR1039512
</td>
<td style="text-align:left;">
untrt
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR1039516
</td>
<td style="text-align:left;">
SRR1039516
</td>
<td style="text-align:left;">
untrt
</td>
</tr>
<tr>
<td style="text-align:left;">
SRR1039520
</td>
<td style="text-align:left;">
SRR1039520
</td>
<td style="text-align:left;">
untrt
</td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
<div id="de-gene-summary" class="section level2">
<h2>DE gene summary</h2>
<table class=" lightable-classic-2" style="font-family: &quot;Arial Narrow&quot;, &quot;Source Sans Pro&quot;, sans-serif; width: auto !important; margin-left: auto; margin-right: auto;">
<caption>
DE gene summary (FDR cutoff = 0.1)
</caption>
<thead>
<tr>
<th style="text-align:left;">
Contrast
</th>
<th style="text-align:right;">
Num_up_genes
</th>
<th style="text-align:right;">
Num_down_genes
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
trt_vs_untrt
</td>
<td style="text-align:right;">
1872
</td>
<td style="text-align:right;">
1481
</td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
<div id="method-summary" class="section level2">
<h2>Method summary</h2>
<p>Adaptor sequences and low-quality bases were trimmed from RNA-seq
reads using Trim Galore. Trimmed reads were then aligned to refernece
genome using STAR with the “–quantMode GeneCounts” parameter
enabled.</p>
<p>Raw count data were input into DESeq2, where normalization was
performed and dispersion estimates were then computed, and a generalized
linear model was fit to test for differential expression. P-values were
adjusted for multiple testing using the Benjamini–Hochberg method to
control the false discovery rate (FDR). Genes with an adjusted p-value
below 0.1 (default setup, you can change it in the config.yaml) were
considered significantly differentially expressed. For more details
about DESeq2 algorithms, please see <a
href="https://www.bioconductor.org/packages/devel/bioc/vignettes/DESeq2/inst/doc/DESeq2.html">DESeq2
vignette</a>.</p>
<p>Gene set enrichment and pathway enrichment analyses and
visualizations are performed using the R Bioconductor package
clusterProfiler. For more details about ClusterProfiler, please see <a
href="https://yulab-smu.top/biomedical-knowledge-mining-book/">clusterProfiler
vignette</a></p>
<p><br></p>
</div>
<div id="software-versions-and-references"
class="section level2 unnumbered">
<h2 class="unnumbered">Software versions and references</h2>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-felix_trim_galore_v0.6.10" class="csl-entry">
Krueger, Felix, Frankie James, Phil Ewels, Ebrahim Afyounian, Michael
Weinstein, Benjamin Schuster-Boeckler, Gert Hulselmans, and Sclamons.
2023. <span>“FelixKrueger/TrimGalore: V0.6.10 - Add Default
Decompression Path.”</span> Zenodo. <a
href="https://doi.org/10.5281/ZENODO.7598955">https://doi.org/10.5281/ZENODO.7598955</a>.
</div>
<div id="ref-love_moderated_2014" class="csl-entry">
Love, Michael I., Wolfgang Huber, and Simon Anders. 2014.
<span>“Moderated Estimation of Fold Change and Dispersion for
<span>RNA</span>-Seq Data with <span>DESeq</span>2.”</span> <em>Genome
Biology</em> 15: 550. <a
href="https://doi.org/10.1186/s13059-014-0550-8">https://doi.org/10.1186/s13059-014-0550-8</a>.
</div>
<div id="ref-mccarthy_differential_2012" class="csl-entry">
McCarthy, Davis J., Yunshun Chen, and Gordon K. Smyth. 2012.
<span>“Differential Expression Analysis of Multifactor
<span>RNA</span>-Seq Experiments with Respect to Biological
Variation.”</span> <em>Nucleic Acids Research</em> 40 (10): 4288–97. <a
href="https://doi.org/10.1093/nar/gks042">https://doi.org/10.1093/nar/gks042</a>.
</div>
<div id="ref-yu_cluster_4.0" class="csl-entry">
Wu, Tianzhi, Erqiang Hu, Shuangbin Xu, Meijun Chen, Pingfan Guo, Zehan
Dai, Tingze Feng, et al. 2021. <span>“clusterProfiler 4.0: A Universal
Enrichment Tool for Interpreting Omics Data.”</span> <em>The
Innovation</em> 2 (3): 100141. <a
href="https://doi.org/10.1016/j.xinn.2021.100141">https://doi.org/10.1016/j.xinn.2021.100141</a>.
</div>
<div id="ref-yu_13years_clusterprofiler_2024" class="csl-entry">
Yu, Guangchuang. 2024. <span>“Thirteen Years of clusterProfiler.”</span>
<em>The Innovation</em> 5 (6): 100722. <a
href="https://doi.org/10.1016/j.xinn.2024.100722">https://doi.org/10.1016/j.xinn.2024.100722</a>.
</div>
<div id="ref-yu_omics_2012" class="csl-entry">
Yu, Guangchuang, Li-Gen Wang, Yanyan Han, and Qing-Yu He. 2012.
<span>“clusterProfiler: An r Package for Comparing Biological Themes
Among Gene Clusters.”</span> <em>OMICS: A Journal of Integrative
Biology</em> 16 (5): 284–87. <a
href="https://doi.org/10.1089/omi.2011.0118">https://doi.org/10.1089/omi.2011.0118</a>.
</div>
</div>
</div>

<br><br><br><br><br><p>Generated by VAI BBC.</p>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
